session19/src/challenge2/Entities.java

```diff
@@ -0,0 +1,31 @@
+package challenge2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Entities {
+
+    int id;
+    String name;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Entities entities = (Entities) o;
+
+        return id == entities.id;
+    }
+
+    public static void main(String[] args) {
```

Comment:
it would be a good idea to move this `main` to a separate class responsible only for running the program (respect "Single Responsibility class" from SOLID).